pkg/ingestor/key/key.go

```diff
@@ -0,0 +1,33 @@
+//
+// Copyright 2022 The GUAC Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package key
+
+type KeyType string
+
+type KeyScheme string
+
+/* const (
```

Comment:
Is this commented section supposed to be here?

Reply:
I put that as a reference but will be used by the verifier implementation to specify `KeyType` and `KeyScheme` into the `Identity` struct

Reply:
Should we push in-toto to make this public? @SantiagoTorres ?

Reply:
I will remove this comment for now.